src/arglist.c

```diff
@@ -95,35 +95,35 @@ alist_new(void)
  * If "fnum_list" is not NULL, use "fnum_list[fnum_len]" as a list of buffer
  * numbers to be re-used.
  */
-    void
+void
 alist_expand(int *fnum_list, int fnum_len)
 {
-    char_u	**old_arg_files;
-    int		old_arg_count;
-    char_u	**new_arg_files;
-    int		new_arg_file_count;
-    char_u	*save_p_su = p_su;
-    int		i;
+    char_u **old_arg_files;
+    int old_arg_count;
+    char_u **new_arg_files;
+    int new_arg_file_count;
+    char_u *save_p_su = p_su;
+    int i;
```

Comment:
As mentioned in the other comment, the existing formatting for the variables should be kept.  The variable names should be aligned using a softtab.

Reply:
When should tabs be used and when should spaced be used?

Reply:
The following indentation settings are used in the Vim source code:

tabstop=8
shiftwidth=4
softtabstop=4

Spaces are used for indentation.  If the number of spaces is equal to a tab width (8),
then a tab character is used instead of 8 spaces.  This behavior is described in the
help for 'softtabstop'.  The idea is to reduce the number of characters used for
indentation.

Reply:
Wait, you indent with spaces but align with tabs? Cursed.

Reply:
> The idea is to reduce the number of characters used for
indentation.

Why is this desirable?

Reply:
This is done to reduce the size of the source files.  Note that this convention was followed from the time when storage space was expensive.